config/jobs/kubernetes/sig-testing/apidiff.yaml

```diff
@@ -27,7 +27,7 @@ presubmits:
         args:
         - /bin/sh
         - -c
-        - "./hack/apidiff.sh -r ${PULL_BASE_SHA}"
+        - "git checkout ${PULL_PULL_SHA} && ./hack/apidiff.sh -r ${PULL_BASE_SHA}"
```

Comment:
Perhaps there is a better way, like not merging in the first place, but I don't know how to configure that.